components/backup-stream/src/endpoint.rs

```diff
@@ -879,24 +903,33 @@ where
             let mts = self.prepare_min_ts().await;
             let sched = self.scheduler.clone();
             info!("min_ts prepared for flushing"; "min_ts" => %mts);
+            let (tx, _) = tokio::sync::mpsc::channel(1);
             self.region_op(ObserveOp::ResolveRegions {
                 callback: Box::new(move |res| {
-                    try_send!(sched, Task::ExecFlush(task, res));
+                    try_send!(sched, Task::ExecFlush(task, res, tx));
                 }),
                 min_ts: mts,
             })
             .await
         })
     }
 
-    fn on_exec_flush(&mut self, task: String, resolved: ResolvedRegions) {
+    fn on_exec_flush(&mut self, task: String, resolved: ResolvedRegions, cb: Sender<FlushResult>) {
```

Comment:
where can we know there is force flush occurs? from tikv log or metrics?

Reply:
For now we can get the `ForceFlush` tasks in the metric `tikv_log_backup_interal_actor_acting_duration_sec_count`.